src/ViewModel/SeeMoreLink.php

```diff
@@ -15,11 +15,12 @@ final class SeeMoreLink implements ViewModel
 
     protected $url;
 
-    public function __construct(Link $link, bool $isInline = false)
+    public function __construct(Link $link, bool $isInline = false, string $ariaLabel = null)
```

Comment:
I think we should move the ariaLabel to the second parameter.

But also, seeing it like this does make me wonder whether we should incorporate ariaLabel directly into the link which could mean that we gain much wider support!